src/content/articles/local/test-id.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const article = {
+    id: 'test-id',
+    title: 'test-article',
+    content: <div>idk</div>,
+}
```

Comment:
React has a neat ability to store JSX within an object, might help us out

Also, maybe we should use some typescript to standardize what an article object looks like across the app